assets/js/themes.js

```diff
@@ -48,7 +69,7 @@ function update_mode (mode) {
 		// If preference for light mode indicated
 		set_mode(false);
 	}
-    else if (document.getElementById('dark_mode_pref').textContent === '' && window.matchMedia('(prefers-color-scheme: dark)').matches) {
+    else if (document.getElementById('dark_mode_pref').textContent === '') {
```

Comment:
Is there a specific reason for removing this line? Currently, this will set the default to dark mode in all cases where no preference is indicated (regardless of `prefers-color-scheme`).

Reply:
Mostly because it serves no purpose with the new watch event. But I can add it back if you still want it.